.circleci/config.yml

```diff
@@ -30,10 +30,7 @@ jobs:
               --older-than 1h \
               --force \
               --config ./.circleci/nuke_config.yml \
-              --exclude-resource-type iam \
-              --exclude-resource-type transit-gateway \
-              --exclude-resource-type transit-gateway-attachment \
-              --exclude-resource-type transit-gateway-route-table
```

Comment:
I think we were excluding transit gateway because it wasn't enabled in some regions and cloud-nuke wasn't handling that correctly. I believe that was fixed in https://github.com/gruntwork-io/cloud-nuke/pull/188, so removing the exclusion is the right thing to do.

Reply:
Ahhh makes sense! Thanks for the clarification!